### DISCUSSION TITLE
I am giving you Linux OS Configuration projucer files

### DISCUSSION THREAD

**Discussion by tomasmark79** [andrewreeman/SpectralSuite#77] [Ideas]
Hello,

I wanted doing pull request but no permission so I am writting here.

I did Linux OS Projucer configuration and I want to share all those files with you.

All the projucer files are available to download from my repo which I sucesfuly compiled.

link: https://github.com/tomasmark79/SpectralSuite-2.1.12

I hope as the template will be enough for you. 

Enjoy.

**Comment by andrewreeman** [comment]
Hi @tomasmark79 , that's fantastic I appreciate you adding this! I'll check this out and see if I can integrate it ok 👍
